jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -1609,6 +1610,50 @@ ecma_builtin_array_prototype_object_unshift (const ecma_value_t args[], /**< arg
   return ecma_make_number_value (new_len);
 } /* ecma_builtin_array_prototype_object_unshift */
 
+/**
+ * The Array.prototype object's 'at' routine
+ *
+ * See also:
+ *          ECMA-262 Stage 3 Draft
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+ecma_builtin_array_prototype_object_at (const ecma_value_t index, /**< index argument */
+                                        ecma_object_t *obj_p, /**< object */
+                                        ecma_length_t len) /**< object's length */
+{
+
+  ecma_number_t relative_index, k;
+  ecma_value_t index_to_integer = ecma_op_to_integer (index, &relative_index);
```

Comment:
The "index_to_integer" is a bit misleading, maybe we could use another name for this? Maybe `conversion_result` (not sure if it is a good name)?